var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -175,17 +178,18 @@ def cmake_args(self):
         args.extend(
             [
                 self.define("EXAGO_ENABLE_GPU", "+cuda" in spec or "+rocm" in spec),
+                self.define("PETSC_DIR", spec["petsc"].prefix),
+                self.define("EXAGO_RUN_TESTS", self.run_tests),
```

Comment:
@alalazo I think this is what you were suggesting.

To clarify usage, when running `spack install`, by default `self.run_tests` is False (per [docs](https://spack.readthedocs.io/en/latest/spack.html#spack.package_base.PackageBase.run_tests)).

So, to enable this, one would run `spack install --test root` or `spack install --test all`, and then we would be able to build with testing enabled? I imagine in the case where we have run `spack install`, we would then need to invoke `spack test` to run the tests...

Note that I am still working on testing per #35751

Reply:
@cameronrutherford Thanks, this is exactly what I was thinking. On the test process it's correct what you say, as far as I know. Once stand-alone tests are implemented you can run them at any time using `spack test`